sdcm/cluster.py

```diff
@@ -1992,6 +1992,22 @@ def download_scylla_manager_repo(self, scylla_repo):
         if not self.is_rhel_like():
             self.remoter.run(cmd="sudo apt-get update", ignore_status=True)
 
+    @retrying(n=30, sleep_time=15, allowed_exceptions=UnexpectedExit)
+    def install_package(self, package_mgr, package_name):
+        wait.wait_for(func=self.is_apt_lock_free, step=30, timeout=60, text='Checking if package manager is free')
+        self.remoter.sudo(f'{package_mgr} install -y {package_name}')
+
+    def is_apt_lock_free(self):
+        if self.distro.is_ubuntu:
```

Comment:
following the above comment, this condition can be removed. BTW, it may be a similar case in future for other Deb distors.

Reply:
fixed.